student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,11 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+   // TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+   char username[11]="\0";
+   char username2[11]="\0";
+   char my_name[11] = "saloni1307";
+   *username = *my_username();
+   *username2 = *malloc_username_from_conf_file();
+   TEST_ASSERT_EQUAL_STRING_MESSAGE(username, username2, my_name);
```

Comment:
@saloni1307 use the message you want to print on failure here instead of "my_name", so something like "User name mismatch!" or similar.

Reply:
@saloni1307 you should free the malloc'd string here.